tools/e2etesting/pipeline_standalone.yml

```diff
@@ -3,6 +3,14 @@ name: $(Date:yyyyMMdd)$(Rev:rr)
 trigger: none
 pr: none
 
+schedules:
```

Comment:
Do we need to have a dependency on `Azure.Industrial-IoT` pipeline here similar to `pipeline_orchestrated.yml` ? If no, then I would argue that `pipeline_orchestrated.yml` pipeline should not have it either. There should be a symmetry between them in this regard.